src/README.txt

```diff
@@ -4,76 +4,92 @@ GeoServer Development requires Java 8, Maven, and git. Due to subtle changes in
 
 
 Further readings:
-  http://docs.geoserver.org/latest/en/developer/index.html
+  https://docs.geoserver.org/latest/en/developer/index.html
 
-== Linux ==
+
+Linux
+-----
 
 1) Install prerequisites
 
 Obtain OpenJDK 8, Maven and git from your Linux distribution.
 
 
-== macOS ==
+macOS
+-----
 
 1) Install Java Runtime Environment
 
-Download and install Java 8 runtime environment, as provided by AdoptOpenJDK macOS installers.
-https://adoptopenjdk.net
+   Download and install Java 8 runtime environment, as provided by adoptium macOS installers:
+   https://adoptium.net
+
+   Update your shell environment with:
 
-Update your shell environment with:
-  export JAVA_HOME=/Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home
+     export JAVA_HOME=/Library/Java/JavaVirtualMachines/adoptiumjdk-8.jdk/Contents/Home
   
 The system /usr/bin/java makes use of JAVA_HOME setting above.
 
-2) Download and install git
-https://git-scm.com/download/mac
+2) Download and install git:
+   https://git-scm.com/download/mac
 
-3) Download and install Maven 3
-http://maven.apache.org/download.html
+3) Download and install Maven:
+   https://maven.apache.org/download.html
 
-== Windows ==
+Windows
+-------
 
 1) Install Java Runtime Environment
-Download and install Java 8 runtime environment, as provided by AdoptOpenJDK windows installers.
-https://adoptopenjdk.net
 
-Update Windows *Environment Variables*:
+   Download and install Java 8 runtime environment, as provided by Adoptium windows installers:
+   https://adoptium.net
 
+   Update Windows *Environment Variables*:

```

Comment:
I think the MSI offers to do this for you. Its probably OK as is, and I will provide a follow-up PR if appropriate after further testing.